docker/red-mongodb/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3.7-slim-stretch
+
+WORKDIR /app
+ENV LD_LIBRARY_PATH /usr/local/lib:/usr/lib
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y --no-install-recommends git ca-certificates xz-utils libssl-dev build-essential && \
```

Comment:
After the build we can remove build essentials if they are not needed on run time and they get removed in the same RUN command.